### DISCUSSION TITLE
Question after upgrade from 0.6-DEV to latest 1.5.0

### DISCUSSION THREAD

**Discussion by xFFF6** [Part-DB/Part-DB-server#312] [Q&A] [answered]
1st of all: Thanks to Jan for all his efforts to keep this very nice project running and even getting better.

FYI
My way of the upgrade was just an export of all my parts into CSV and using the given example import file for editing it accordingly. Then I have just re-installed my Raspi with latest Raspi OS / requirements / latest Part-DB and imported all parts with adjusted storage locations.

I already noticed that most of configs are now done via config files, what is no issue for me.
But now my question:

In the older Part-DB Config I was able to check mark an option what displayed the full path of a parts storage location:

![image](https://github.com/Part-DB/Part-DB-server/assets/80603288/bc606193-bd15-441f-80bc-a1042fc1ef62)

Even if it wasn't check marked, the full path was shown when the mouse hovered over it in a little pop up.
Since our little retro group started to merge all our data into one new Part-DB Server, this option is now missing, since we want to have a quick look, who from us is owning a needed part with no need to open up the part details.

I know its not a major issue, but I'm wondering if I'm just missing the correct place to edit or if this option just is "gone". If it's gone, I'd like to request to get it back :)

Thanks and regards
Chris

**Comment by jbtronics** [comment, marked as answer]
Since Part-DB 1.5.2 there now is a env setting `TABLE_DEFAULT_PAGE_SIZE` which allows you to configure the default page size. Set this to `-1` to show all parts by default without pagination.

I also added that the full path of an element in the tables are shown on hover. However, you currently can not set that full paths are shown in the table directly like before.

A long term plan is to add a better configuration system to Part-DB, so that you can change many settings via the WebUI again (and per user).

**Reply by xFFF6** [reply]
Thanks!
